{
  "comments": [
    {
      "unresolved": true,
      "key": {
        "uuid": "c180c7f4_ac6dd003",
        "filename": "/COMMIT_MSG",
        "patchSetId": 1
      },
      "lineNbr": 14,
      "author": {
        "id": 1470474
      },
      "writtenOn": "2021-11-02T15:16:00Z",
      "side": 1,
      "message": "i think we probably want to understand what fence error was resulting as well",
      "range": {
        "startLine": 14,
        "startChar": 0,
        "endLine": 14,
        "endChar": 34
      },
      "revId": "fce89c09631ee13b1356d7bb302998fcb7bb4548",
      "serverId": "85c56323-6fa9-3386-8a01-6480fb634889"
    },
    {
      "unresolved": true,
      "key": {
        "uuid": "0771b131_5edcd010",
        "filename": "/COMMIT_MSG",
        "patchSetId": 1
      },
      "lineNbr": 25,
      "author": {
        "id": 1470474
      },
      "writtenOn": "2021-11-02T15:16:00Z",
      "side": 1,
      "message": "there should be a bug filed + tests describing how this change was tested\n\nalso this Tracked-On clause is empty so it\u0027s not meaningful",
      "range": {
        "startLine": 25,
        "startChar": 0,
        "endLine": 25,
        "endChar": 11
      },
      "revId": "fce89c09631ee13b1356d7bb302998fcb7bb4548",
      "serverId": "85c56323-6fa9-3386-8a01-6480fb634889"
    }
  ]
}